external/source/exploits/cve-2015-0058/cve-2015-0058/LinkCursorPoc.h

```diff
@@ -0,0 +1,70 @@
+typedef unsigned __int64 QWORD;

```

Comment:
On x64 the `ULONG_PTR` type already matches this ( see [here](https://msdn.microsoft.com/en-us/library/windows/desktop/aa383751%28v=vs.85%29.aspx) ).

Can we please use this instead? In the case of x86 (when we get around to making that magic happen) I'm guessing that the values will be required to be `4` bytes anyway, right?